bundle/manifests/argoproj.io_notificationsconfigurations.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.17.2
```

Comment:
Is controller-gen upgrade required? If so, we will also need to update it in makefile
https://github.com/argoproj-labs/argocd-operator/blob/58532cebff27bc97f94d9671bf1cf7fbe4d4906e/Makefile#L169-L171

Reply:
Yes, IIRC, this change was required for the compilation to succeed. Let me cross check and revert if not required.